src/libs/actions/Report.ts

```diff
@@ -954,6 +964,12 @@ function openReport(
         parentReportActionID,
     };
 
+    // temporary flag will be removed once ReportScreen supports MoneyRequestReportView - https://github.com/Expensify/App/issues/57509
+    // Permission will be removed before the end of the project
```

Comment:
Makes sense, we have to keep this in place as we have two report views to support temporarily